### ISSUE TITLE
Multiselect option should persist across sessions

### ISSUE THREAD

**Issue by EasyRhin0** [files-community/Files#6958]
### What's the Problem?

Not providing multiselect as a persistent setting is an accessibility issue. Users like me who only have the use of one hand cannot multiselect using shirt and/or ctrl. The current multiselect feature does not persist across sessions.

### Solution/Idea

Add a setting in Settings > Files & Folders that enables persistent multiselect. The current multiselect checkbox should be disabled when the persistent setting is enabled.

### Alternatives

The proposed solution follows the proven Windows approach.

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow one-handed users to accomplish multiple file/folder selection without repeatedly enabling the setting | Must |

### Files Version

2.0.34.0

### Windows Version

Windows 11 22000.318

**Comment by Eldenroot** [comment]
I cannot find this option anymore... how to enable multiselect via checkboxes please?

**Comment by Orevin** [comment]
I would like to see this option as well

**Comment by yaichenbaum** [comment]
Merging with #8918
